src/main/java/com/flowingcode/vaadin/addons/shareeasy/util/Options.java

```diff
@@ -0,0 +1,76 @@
+/*-
+ * #%L
+ * Share Easy Add-on
+ * %%
+ * Copyright (C) 2023 Flowing Code
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package com.flowingcode.vaadin.addons.shareeasy.util;
+
+import java.util.Optional;
+import org.apache.commons.lang3.StringUtils;
+import com.flowingcode.vaadin.addons.shareeasy.enums.Driver;
+import com.flowingcode.vaadin.addons.shareeasy.enums.Locale;
+import elemental.json.Json;
+import elemental.json.JsonArray;
+import elemental.json.JsonObject;
+
+/**
+ * Representation of the default options that can be configure for a Share Easy component.
+ */
+public class Options {
+
+  /* By default current page's title */
+  public String shareText;
+
+  /* By default current page's url */
+  public String shareLink;
```

Comment:
The same observation applies https://github.com/FlowingCode/ShareEasy/pull/1#discussion_r1302133580

Reply:
Done.